### PR TITLE
DOCS-608 Reintroduce Viridian Content and Update Product Name

### PR DESCRIPTION
**Pull request by angelasimms**
Reintroduces hidden Viridian content and updates product name to Viridian Cloud in line with new branding. Once this PR is approved I can update previous versions 5.3.x and 5.2.x with the same changes.

**Outstanding question:**

On both the configuration page (configuration.adoc) and in Get Started (get-started.adoc) we mention downloading the sample Go client to discover a Viridian cloud cluster. Not sure if an alternative to this has been planned.

### CODE REVIEW

**Review by yuce — APPROVED** [review]
Looks good!

**Comment by angelasimms** [comment]
> Looks good!

Hi @Yuce, I have now updated the Getting Started. If you are happy with this, I'll merge.

File: docs/modules/ROOT/pages/clc-job.adoc [hunk 1]
```diff
@@ -25,7 +25,7 @@ clc job [command] [options]
 
 Creates a Jet job using the provided Jar file.
 
-This command requires a Hazelcast cluster of 5.3.0-BETA-2 or better.
+This command requires a {hazelcast-cloud} or Hazelcast cluster of 5.3.0-BETA-2 or better.
```
Comment (yuce) [inline docs/modules/ROOT/pages/clc-job.adoc:28]:
```suggestion
This command requires a {hazelcast-cloud} or Hazelcast cluster of 5.3.0 or better.
```

**Review by yuce — COMMENTED** [review]
Just two ideas,,.

File: docs/modules/ROOT/pages/clc-job.adoc (hunk 1 above)
Comment (angelasimms) [reply docs/modules/ROOT/pages/clc-job.adoc]: Updated as requested.

File: docs/modules/ROOT/pages/get-started.adoc [hunk 2]
```diff
@@ -1,189 +1,147 @@
...
+Having separate local configurations for your two production clusters allows you to switch between them without leaving the command line. Make sure both clusters are running before you start.
 
-. Execute the following command to use the `dev` configuration to connect to your development cluster:
+. Execute the following command to use the local configuration to connect to your Test1 cluster. Replace the placeholder `$CLUSTER_ID` with the ID of your cluster. 
 +
-[source, bash]
+[source, shell]
 ----
-clc -c dev
+clc -c pr-$CLUSTER_ID
```
[the remark this answers is not in the document]
Comment (angelasimms) [reply docs/modules/ROOT/pages/get-started.adoc]: Updated as requested.

**Review by yuce — APPROVED** [review]
Looks good!

**Review by yuce — DISMISSED** [review]
Looks good!

**Review by yuce — APPROVED** [review]

**Pull request merged**